test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -650,7 +672,7 @@ def forward(self, x):
                 return x.new_zeros(x.data.size())
 
         x = torch.randn(3, 4)
-        self.run_test(Data(), x)
+        self.run_test(Data(), x, remained_onnx_input_idx=[])
```

Comment:
perhaps we should add `dynamic_axes` argument to these calls. That should preserve previous behavior